src/styles/main.scss

```diff
@@ -6,4 +6,5 @@
 @import './global.scss';
 
 // Components
-@import './navbar.scss';
\ No newline at end of file
+@import './navbar.scss';
+@import './countdown.scss';
```

Comment:
is our standard for .scss files going to be importing them into main rather than individually importing into component? if so, could we note that somewhere in style guide :)

Reply:
Yeah definitely! Do you have a preference on which way we make the standard? Would it make more sense to just import them into the component?

Reply:
hmm I don't rly have a preference, not very familiar with styling things. Whatever you / everyone wants as long as we set a standard!